spring-content-cmis/src/main/java/org/springframework/content/cmis/CmisFileName.java

```diff
@@ -0,0 +1,13 @@
+package org.springframework.content.cmis;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Target({METHOD, FIELD})
+@Retention(RUNTIME)
+public @interface CmisFileName {
```

Comment:
SCR already has an annotation for saving the filename `@OriginalFileName`.  

Typically, we map cmis properties to spring content/data annotations where possibe.  What do you think to using that instead of introducing a new `@CmisFileName`

Reply:
In addition, I think we then need to update the relevant field when setting the content like we do with the MimeType [here](https://github.com/paulcwarren/spring-content/blob/99cf8e2eda2bce48ead7c4f197a1d5c975b7eec1/spring-content-cmis/src/main/java/internal/org/springframework/content/cmis/CmisServiceBridge.java#L419).